Study/Forecast.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+namespace Study
+{
+    public class Forecast
+    {
+        private DateTime date;
+        private int temperatureMax = 20;
+        private int temperatureMin;
+        private int pressure = 740;
+        private int humidity = 50;
+        private int windStrength = 2;
+        private EDirection windDirection = EDirection.North;
+        private EVariantsPrecipitation precipitation;
+
+        public Forecast(DateTime date, int temperatureMax, int temperatureMin, int pressure, int humidity, int windStrength, EDirection windDirection, EVariantsPrecipitation precipitation)
+        {
+            this.date = date;
+            SetTemperature(temperatureMax, temperatureMin);
+            SetPressure(pressure);
+            SetHumidity(humidity);
+            SetWind(windStrength, windDirection);
+            SetPrecipitation(precipitation);
+        }
+        public void SetTemperature(int tMax, int tMin)
+        {
+            if (tMax >= -267 && tMax <= 50)
+                temperatureMax = tMax;
+            else
+                Console.WriteLine($"The maximum temperature value is out of range (-267) - 50C. Maximum temperature is set {temperatureMax}.");
+            if (tMin <= this.temperatureMax)
+                this.temperatureMin = tMin;
+            else
+            {
+                this.temperatureMin = temperatureMax - 5;
+                Console.WriteLine($"The minimum temperature exceeds the maximum. The minimum temperature is set {temperatureMin}.");
+            }
+
+        }
+        public void SetPressure(int setPressure)
+        {
+            if (setPressure > 735 && setPressure < 750)
+                this.pressure = setPressure;
+            else
+                Console.WriteLine($"The pressure value is out of range 735 - 750mmHg. Pressure is set = {this.pressure}mmHg.");
+        }
+        public void SetHumidity(int setHumidity)
+        {
+            if (setHumidity >= 0 && setHumidity <= 100)
```

Comment:
Диапазон значений рекомендую обозначать чуть по-другому(эт рекомендация)
```suggestion
            if (0 <= Humidity && Humidity <= 100)
```
так сразу видно, что находится внутри диапазона [0,100]